hfinger/uri_reader.py

```diff
@@ -102,11 +102,14 @@ def uri_fingerprint(pkt):
     if it < 0:
         # method has 7 chars, so no need to cut t1
         meth = t1
+    if "/" not in t:
+        # Simple URI presence check
+        return "|||||||"
     if meth in METHODS:
         # searching for URL, which should be separated from method with whitespace
         it1 = t.find(" ")
         # cutting out URL
-        uri = t[it1 + 1 :]
+        uri = t[it1 + 1:]
```

Comment:
Akurat po usunięciu tej spacji wydaje mi się to mniej czytelne niż było :thinking:
To Ci podpowiedział jakiś linter?
(Podkreślam, że to jest moja subiektywna opinia)

Reply:
Something happened between commits - now fixed in 9ef4960628091064a0287f8da80bdcc408a471c7.